### PR TITLE
Update RUM schema to latest

### PR DESCRIPTION
**Pull request by fuzzybinary**
### What does this PR do?

Update RUM schema to latest to add new Error sources.

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: features/dd-sdk-android-rum/src/main/json/rum/_common-schema.json [hunk 1]
```diff
@@ -41,6 +41,11 @@
       "description": "The build version for this application",
       "readOnly": true
     },
+    "build_id": {
```
Comment (0xnm) [inline features/dd-sdk-android-rum/src/main/json/rum/_common-schema.json:44]: let's remove this one, this https://github.com/DataDog/dd-sdk-android/pull/1756 covers it

File: features/dd-sdk-android-rum/src/main/json/rum/vital-schema.json
Comment (0xnm) [inline features/dd-sdk-android-rum/src/main/json/rum/vital-schema.json:1]: we maybe can drop this file for now, we can address this change later when it is needed, with a proper setup of fake data for it.

**Review by xgouchet — APPROVED** [review]

**Review by jonathanmos — APPROVED** [review]

File: features/dd-sdk-android-rum/src/main/json/rum/_common-schema.json (hunk 1 above)
Comment (fuzzybinary) [reply features/dd-sdk-android-rum/src/main/json/rum/_common-schema.json:44]: I'll wait and rebase once #1756 goes in.  I can work off of this branch to start work on the source override.

**Comment by fuzzybinary** [comment]
Closing in favor of #1855

**Pull request closed without merging**